experimental/LieAlgebras/docs/src/introduction.md

```diff
@@ -16,7 +16,7 @@ This part of OSCAR is in an experimental state; please see [Adding new projects
 
 Please direct questions about this part of OSCAR to the following people:
 * [Lars Göttgens](https://lgoe.li/)
-* [Laura Voggesberger](https://www.ruhr-uni-bochum.de/ffm/Lehrstuehle/Lehrstuhl-VI/voggesberger.html)
+* [Felix Röhrich](https://www.art.rwth-aachen.de/cms/~xlgua)
```

Comment:
While looking through the existing documentation, I noticed that this contact section is a bit outdated.
Is it fine for you (@felix-roehrich) to be listed here (and then similarly after moving things to src in the contact section there)?
In the `src/` documentation, I would find it beneficial to have someone more permanent than Felix and me there, and would additionally put @fingolfin (if that's fine for you).

Reply:
Fine by me